Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -1467,7 +1467,7 @@ CookieConsentAPIEnabled:
 # FIXME: This is handled via WebView SPI rather than WebPreferences for WebKitLegacy. We should change the SPI to lookup the WebPreferences value instead.
 CookieEnabled:
   type: bool
-  status: developer
+  status: embedder
```

Comment:
Why is this an "embedder" feature. There is a Safari setting for this, isn't there?

Reply:
This was previously a WebPreference, without UI. I thought we might find it useful to toggle as a web developer, but since it's already exposed through Safari UI I moved it back to 'embedder' which acts like 'WebPreference'. I should not have ever made it 'developer'.